src/main.rs

```diff
@@ -69,5 +68,16 @@ fn event_handler(handle: Handle, keycode: Option<char>, modifiers: KeyModifier)
 
 fn main() {
     env_logger::init();
-    run_event_listener(&event_handler);
+
+    let win = WindowDesc::new(ui::main_ui_builder)
+        .title("gõkey")
+        .window_size((340.0, 210.0));
+    let app = AppLauncher::with_window(win);
+    let event_sink = app.get_external_handle();
```

Comment:
Instead of passing event sink through to keyboard event loop, create a static var `Lazy<ExtEventSink>` and init that static var here.

Reply:
✅ good idea, updating...

Reply:
Updated